Add Error suffix for custom exceptions

**Describe the bug**
Recently in #424 , `flake` pointed out issues in naming conventions. Because of that, I am creating this issue as a /kind todo , because the issue is not relevant in the mentioned PR 

**LOGS**
```
 [INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8:pep8-naming.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Tests should end in _test.py.........................(no files to check)Skipped
Check for added large files..............................................Passed
Check for byte-order marker..............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check JSON...............................................................Passed
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
Check python ast.........................................................Passed
Debug Statements (Python)................................................Passed
pydocstyle...............................................................Passed
Check Toml...............................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
mypy.....................................................................Passed
black....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1
srcopsmetrics/exceptions.py:21:8: N818 exception name 'MissingPreviousKnowledge' should be named with an Error suffix
srcopsmetrics/exceptions.py:25:8: N818 exception name 'NotKnownEntities' should be named with an Error suffix
```

--- a/srcopsmetrics/bot_knowledge.py
+++ b/srcopsmetrics/bot_knowledge.py
@@ -25,7 +25,7 @@ from pkgutil import iter_modules
 from typing import List, Optional, Tuple
 
 from srcopsmetrics.entities import Entity, NOT_FOR_INSPECTION
-from srcopsmetrics.exceptions import NotKnownEntities
+from srcopsmetrics.exceptions import NotKnownEntitiesError
 from srcopsmetrics.github_knowledge import GitHubKnowledge
 from srcopsmetrics import utils
 
@@ -80,7 +80,9 @@ def analyse_projects(
         if entities:
             specified_entities = [e for e in allowed_entities if e.__name__ in entities]
             if specified_entities == []:
-                raise NotKnownEntities(message="", specified_entities=entities, available_entities=allowed_entities)
+                raise NotKnownEntitiesError(
+                    message="", specified_entities=entities, available_entities=allowed_entities
+                )
 
         inspected_entities = specified_entities or allowed_entities
 

--- a/srcopsmetrics/exceptions.py
+++ b/srcopsmetrics/exceptions.py
@@ -18,11 +18,11 @@
 """Exceptions for SrcOpsMetrics."""
 
 
-class MissingPreviousKnowledge(Exception):
+class MissingPreviousKnowledgeError(Exception):
     """An exception when no previous knowledge has been extracted for a project."""
 
 
-class NotKnownEntities(Exception):
+class NotKnownEntitiesError(Exception):
     """An exception when Entities requested are not known."""
 
     def __init__(self, message, specified_entities, available_entities):

